features/AssetCategory.feature

```diff
@@ -284,4 +283,102 @@ Feature: AssetCategory
+    When I successfully execute the action "device-manager/assetCategory":"create" with args:
+      | engineId  | "engine-ayse"  |
+      | body.name | "coloredTruck" |
+    When I successfully execute the action "device-manager/metadata":"create" with args:
+      | engineId       | "engine-ayse"          |
+      | body.name      | "color"                |
+      | body.valueType | "string"               |
+      | body.valueList | ["red","blue","green"] |
+      | body.mandatory | true                   |
+    When I successfully execute the action "device-manager/assetCategory":"linkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+    When I successfully execute the action "device-manager/assetCategory":"unlinkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+    When I successfully execute the action "device-manager/assetCategory":"linkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+      | body.value | "red"          |
+    When I successfully execute the action "device-manager/assetCategory":"unlinkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+    When I execute the action "device-manager/assetCategory":"linkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+      | body.value | "black"        |
+    Then I should receive an error matching:
+      | status | 400 |
+
+
+  Scenario: Use metadata enum type link with assetCategory, and create asset with
+    When I successfully execute the action "device-manager/assetCategory":"create" with args:
+      | engineId  | "engine-ayse" |
+      | body.name | "typedTruck"      |
+    When I successfully execute the action "device-manager/metadata":"create" with args:
+      | engineId       | "engine-ayse"   |
+      | body.name      | "truckType"     |
+      | body.valueType | "string"        |
+      | body.valueList | ["big","small"] |
+      | body.mandatory | true            |
+    When I successfully execute the action "device-manager/assetCategory":"linkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "typedTruck" |
+      | metadataId | "truckType"        |
+    When I execute the action "device-manager/asset":"create" with args:
+      | engineId                | "engine-ayse" |
+      | body.type               | "truck"       |
+      | body.model              | "M"           |
+      | body.reference          | "asset_02"    |
+      | body.category           | "typedTruck"  |
+      | body.metadata.truckType | "super"       |
+    Then I should receive an error matching:
+      | status | 400 |
```

Comment:
Same comment about matching the `error.id` property

Reply:
Done